js-storefront/bootstrap/patches/spartacus-configuration.module.ts.patch

```diff
@@ -11,7 +11,7 @@
 +// <meta name="occ-backend-base-url" > is ignored.
 +// This in turn breaks the deployment in CCv2.
 +// https://github.com/SAP/spartacus/issues/5886
-+if (environment.occBaseUrl) {
++if (environment.occBaseUrl && occConfig && occConfig.backend && occConfig.backend.occ) {
```

Comment:
not the prettiest code, but as it solves an compiler warning for a workaround that will be removed in the future, I think it is a pragmatic way...